### DISCUSSION TITLE
Collabora CODE Errors/Documentation Expected(?)

### DISCUSSION THREAD

**Discussion by Snuupy** [nextcloud/all-in-one#4874] [Questions] [answered]
Hi,

CODE works, but errors show up in logs.

Errors that are expected/are not concerning/showstoppers should be listed so users know which errors are normal to see. On a new installation, I see several errors that are not show stoppers. These include:

```
[ forkit ] WRN  The systemplate directory [/opt/cool/systemplate] is read-only, and at least [/opt/cool/systemplate//etc/hosts] is out-of-date. Will have to copy sysTemplate to jails. To restore optimal performance, make sure the files in [/opt/cool/systemplate/etc] are up-to-date.| common/JailUtil.cpp:432
```
and
```
sh: 1: /usr/bin/coolmount: Operation not permitted
```

Please see these comments:

https://github.com/CollaboraOnline/online/issues/779#issuecomment-740250110
https://github.com/CollaboraOnline/online/issues/779#issuecomment-1501321652

So either there should be an option to enable `cap_add: SYS_ADMIN`, or `cap_add: SYS_ADMIN` should be added to `php/containers.json`, or it should be documented somewhere (possibly here, or in a `.md` file) that these errors are expected and are normal despite negatively affecting performance.

If I specifically wanted to add `cap_add: SYS_ADMIN` to the CODE container, is there a way for me to do so?

Thank you for maintaining this repo!

**Comment by szaimen** [comment, marked as answer]
Hi, thanks for the hint! SYS_ADMIN will be added with https://github.com/nextcloud/all-in-one/pull/4876

**Reply by szaimen** [reply]
This is now released with v9.1.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
